openmetadata-spec/src/main/resources/json/schema/type/changeEvent.json

```diff
@@ -50,5 +50,5 @@
     }
   },
   "required": ["id", "eventType", "entityType", "entityId", "timestamp"],
-  "additionalProperties": false
+  "additionalProperties": true
```

Comment:
we need to think of different way , nobody will remeber a additionalProperty named containing alertName, and that information is already present and can be passed onto the Destination